cmake/configure_files/LBANNConfig.cmake.in

```diff
@@ -49,6 +49,7 @@ set(LBANN_HAS_LBANN_PROTO @LBANN_HAS_LBANN_PROTO@)
 set(LBANN_HAS_OPENCV @LBANN_HAS_OPENCV@)
 set(LBANN_HAS_NCCL2 @LBANN_HAS_NCCL2@)
 set(LBANN_HAS_PROTOBUF @LBANN_HAS_PROTOBUF@)
+set(LBANN_HAS_PYTHON_FRONTEND @LBANN_HAS_PYTHON_FRONTEND@)
```

Comment:
Is this needed? I'd prefer keeping the Python front-end purely downstream from the C++.
```suggestion
```

Reply:
The two are somewhat indistinguishable -- the python front-end is dependent on the `*.proto` files, which are part of the build. This file is not for C++ integration, it's for *CMake* integration, which might do any variety of things. Suppose we are a downstream of LBANN. We might want to know if we can do stuff with they python front-end as part of the CMake configuration of our project. The right answer might be to just try to find python and then try to load the `lbann` module, but I was thinking this is a nearly-zero-overhead way to short-circuit if we don't have the python front-end and don't otherwise need python.

I will more readily agree to your suggestion if there's a way to create the python front-end independent of a specific build of LBANN.

Reply:
Oh, I understand now. I was confusing this with `lbann_config.hpp.in`. This is completely sensible.

Reply:
Cool; then I won't make a change.